pkg/virt-handler/vm.go

```diff
@@ -1466,8 +1462,7 @@ func (c *VirtualMachineController) recordPhaseChangeEvent(vmi *v1.VirtualMachine
 
 func calculatePausedCondition(vmi *v1.VirtualMachineInstance, reason api.StateChangeReason) {
 	now := metav1.NewTime(time.Now())
-	switch reason {
-	case api.ReasonPausedMigration:
+	if isVMIPausedDuringMigration(vmi) {
```

Comment:
It will now add the condition regardless of what `reason` is. Is that what we want?

What about putting something like this into the switch case?

```
if !isVMIPausedDuringMigration(vmi) {
    return
}
```

Reply:
> It will now add the condition regardless of what `reason` is. Is that what we want?
> 
> What about putting something like this into the switch case of `api.ReasonPausedMigration`?
> 
> ```
> if !isVMIPausedDuringMigration(vmi) {
>     return
> }
> ```

Pused a [change](https://github.com/kubevirt/kubevirt/compare/59d33eb4be3447ca062f6770a9cdffc14f30f465..d8a26d1e78c95e22391947047d3137ada25918e5) to also check the reason